Neos.Neos/Configuration/Policy.yaml

```diff
@@ -325,8 +325,7 @@ roles:
   'Neos.Neos:Administrator':
     label: Neos Administrator
     description: Grants access to all modules and functionalities of the Neos backend.
-
-    parentRoles: ['Neos.Neos:Editor']
+    parentRoles: ['Neos.Neos:Editor', 'Neos.Media.Browser:WorkspaceName']
```

Comment:
It would be better to assign the privilege in the Media.Browser package to not add a dependency here.
This is already done there for several other privileges like this example:
```yaml
  'Neos.Neos:Administrator':
    privileges:
      -
        privilegeTarget: 'Neos.Media.Browser:ManageAssetCollections'
        permission: GRANT
```